src/game/coreObjects/Piece.js

```diff
@@ -15,7 +15,7 @@ import { COLLISION_RADIUS } from "../coreState/utils/Params";
 // A single piece in the game, which can move in different directions and detect collisions
 // based on which direction is moving.
 class Piece {
-  constructor(cellType = CELL_TYPE.NORMAL) {
+  constructor(cellType = CELL_TYPE.NORMAL, coreState = undefined) {
```

Comment:
Go through the constructors, it looks like we might not need the default arguments